test/framework/src/test/java/org/opensearch/cluster/service/FakeThreadPoolMasterServiceTests.java

```diff
@@ -84,21 +84,21 @@ public void testFakeClusterManagerService() {
         doAnswer(invocationOnMock -> runnableTasks.add((Runnable) invocationOnMock.getArguments()[0])).when(executorService).execute(any());
         when(mockThreadPool.generic()).thenReturn(executorService);
 
-        FakeThreadPoolMasterService masterService = new FakeThreadPoolMasterService(
+        FakeThreadPoolMasterService clusterManagerService = new FakeThreadPoolMasterService(
```

Comment:
Does the class `FakeThreadPoolMasterService` need to be renamed as well?

Reply:
@kotwanikunal Thanks for your review. I forgot to write a note for this. 😂
I'm not going to rename this class in this PR. I plan to use the way of dealing with other classes on the 3 classes: `FakeThreadPoolMasterService` `BlockMasterServiceOnMaster` `BusyMasterServiceDisruption`.
One PR to rename directly, and one PR to create a new class with the old name then make it a subclass of the class in new name.

Reply:
👍 . Hopefully you have a task to keep track of these 🙂

Reply:
😁 I listed the remaining items in this issue https://github.com/opensearch-project/OpenSearch/issues/3543#issuecomment-1189593243, and I will check it before wrapping up the work.